spec/integration/_dev/deploy/tf/spec.yml

```
@@ -25,3 +25,8 @@ spec:
       type: file
       contentMediaType: "application/json"
       required: false
+    - description: log files with events to ingest
+      name: logs
+      type: folder
+      required: false
+      $ref: "./logs/spec.yml"
```

I wonder if we should add a directory to include any static file that can be used in terraform scenarios, logs or otherwise.
```suggestion
    - description: Files that can be used with the file provisioner of Terraform
      name: files
      type: folder
      required: false
      $ref: "./files/spec.yml"
```

---

Probably yeah... But how would we define the spec file for `files` what extension do we allow?